plugin/settings.gradle.kts

```diff
@@ -2,13 +2,6 @@ pluginManagement {
     val latestRelease = file("VERSION_LATEST_RELEASE.txt").readText().trim()
     plugins {
         id("org.jlleitschuh.gradle.ktlint") version latestRelease
```

Comment:
@JLLeitschuh If you have any idea how we can move it to toml file, let me know.
I bet that it is not possible to read a version from file when using toml file.